src/include/cursor.h

```diff
@@ -93,8 +93,9 @@ struct __wt_cursor_btree {
      * stack (the stack of skiplist entries leading to the insert point). The search functions also
      * return the relationship of the search key to the found key.
      */
-    WT_REF *ref;   /* Current page */
-    uint32_t slot; /* WT_COL/WT_ROW 0-based slot */
+    WT_REF *last_ref; /* Previous page */
```

Comment:
This field is `diagnostic` only,  isn't it?

Reply:
Fixed